src/compiler/transformers/ts.ts

```diff
@@ -2261,7 +2243,7 @@ namespace ts {
 
             const statements: Statement[] = [];
             if (shouldEmitVarForEnumDeclaration(node)) {
-                addVarForEnumDeclaration(statements, node);
+                addVarForEnumOrModuleDeclaration(statements, node);

```

Comment:
I can't find `addVarForEnumOrModuleDeclaration` in `transforms` branch, and I don't see it added here. Am I missing something?